tools/plugins/com.liferay.ide.server.ui/src/com/liferay/ide/server/ui/portal/PortalRuntimeComposite.properties

```diff
@@ -0,0 +1,9 @@
+browse=Browse...
+detectedBundleType=Detected bundle type
+installedJREs=Installed JREs...
+liferayPortalRuntime=Liferay Portal Runtime
+liferayPortalRuntimeDirectory=Liferay Portal Bundle Directory
+name=Name
+selecteRuntimeJRE=Select runtime JRE
+selectLiferayPortalDirectory=Select Liferay Portal directory
+specifyInstallationDirectory=Specify the installation directory of the Portal configured with Liferay.
```

Comment:
Lets change this text to say "Specify the installation directory of the Portal bundle"